plugin.js

```diff
@@ -7,6 +7,12 @@ CKEDITOR.plugins.add( 'bylawlist', {
   icons: 'bylawlist', // Bylaw List icon
   init: function( editor ) {
       editor.addCommand( 'insertBylawlist', {
+        allowedContent: {
```

Comment:
Because of the class this needs to use the object format for the rules. Not the default string format.

https://ckeditor.com/docs/ckeditor4/latest/guide/dev_allowed_content_rules.html#object-format